Assets/Scripts/Networking/PlayerManager.cs

```diff
@@ -280,7 +281,7 @@ public void Update()
                     animatorHands.SetBool("isDown", true);
                 }   
             }
-            if (IsFiring && !waitingToShoot && health > 0f)
+            if (IsFiring && !waitingToShoot && health > 0f && !gameObject.GetComponent<FirstPersonController>().areSettingsEnabled)
```

Comment:
FirstPersonController script is disabled for the other player. Are you sure this works in multiplayer?

Reply:
Perhaps if you want to disable shooting while in the settings menu, you should not allow IsFiring to be set to true? To test, you want the other player to click mouse while in the menu and you should not see him shoot.

Reply:
You were right and I did fix